mysql51/src/main/java/another/PropertyHandler.java

```diff
@@ -0,0 +1,10 @@
+package another;
+
+public class PropertyHandler extends mysql51.PropertyHandler {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Rename this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2176)